katcp/resource.py

```diff
@@ -595,10 +595,15 @@ def parent_name(self):
 
     @property
     def name(self):
-        """Name of this KATCP resource"""
+        """Name of this KATCPSensor"""
         return self._name
 
     @property
+    def python_id(self):
```

Comment:
Another reason to make it "normalise_name" would be that we need to address the sensor name with underscores outside the python code eg. in gui or in the hdf5 files. And I already call it normalise_name in katstore ;)